odxtools/element.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import List, Optional, TypedDict, cast
+from xml.etree import ElementTree
+
+from odxtools.exceptions import odxrequire
+from odxtools.utils import create_description_from_et
+
+from .odxlink import OdxDocFragment, OdxLinkId
+
+
+class BaseElementKwargs(TypedDict):
```

Comment:
make this a non-public class?
```suggestion
class _NamedElementAttribDict(TypedDict):
```

Reply:
(maybe it could also be moved into the `NamedElement` class, a la `NamedElement._AttribDict`)

Reply:
The class was removed